ql/termstructures/volatility/optionlet/constantoptionletvol.hpp

```diff
@@ -35,29 +36,33 @@ namespace QuantLib {
     class ConstantOptionletVolatility : public OptionletVolatilityStructure {
       public:
         //! floating reference date, floating market data
-        ConstantOptionletVolatility(Natural settlementDays,
-                                    const Calendar& cal,
+        ConstantOptionletVolatility(Natural settlementDays, const Calendar &cal,
                                     BusinessDayConvention bdc,
-                                    const Handle<Quote>& volatility,
-                                    const DayCounter& dc);
+                                    const Handle< Quote > &volatility,
```

Comment:
I'm guilty. I used git with clang-format on the commit. 

Wouldn't it make sense to provide a generic clang-format rule file and just use that consistently?